### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by abreham1222**
Added the required files and description to the README.md file
 1. index.html
 2. styles.css
 3. Stylelinters,json

### CODE REVIEW

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi ** @abreham, **

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight 💪🏻

- No linter errors. ✔️ 
- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

- [ ] Kindly, Add a descriptive README file to your project as required please use this [template](https://github.com/microverseinc/readme-template) You can copy past but remember to customize it to your project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+Demo project on Microverse
```
Comment (zainsadaqat) [inline README.md:2]: # Hey @abreham1222 😀 

- [ ] As requested by the previous Code Reviewer, Please add the readme template provided by Microverse. You can find the template [here](https://github.com/microverseinc/readme-template) then copy and paste the content to your README.md file and kindly don't forget to add your Project Name, Description, Steps to run your Project, Your Social Media links, Tech stack used in your project in that README.md file 👍🏻 😄

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hey @abreham1222  😄  

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] As requested by the previous Code Reviewer Please, Write a meaningful title for your pull request and always add a short summary of changes introduced in the pull request. you can check the [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). For example, instead of "Feature Branch" write something meaningful that tells more about what's in the Pull Request.

[Optional]

- [ ] Good job @abreham1222 for writing descriptive commit messages however some of your commit messages are not Imperative. For example, Instead of "_Added new heading to index.html_" use "_Add new heading to index.html_". Please keep this in mind in future projects 😄. If you still have some confusion then please read this article on [How to Write Good Commit Messages: A Practical Git Guide](https://www.freecodecamp.org/news/writing-good-commit-messages-a-practical-guide/) 👍🏻 

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (abreham1222) [reply README.md]: HI, @zainsadaqat I hope I did your request.

**Review by CollinsTatang — APPROVED** [review]
Hi @abreham1222, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**